build/Common.props

```diff
@@ -14,8 +14,6 @@
     <Version Condition=" '$(Version)' == '' ">$(VersionPrefix)$(VersionSuffix)</Version>
     <AssemblyVersion Condition="$(AssemblyVersion) == ''">$(VersionPrefix).0</AssemblyVersion>
     <FileVersion  Condition="$(FileVersion) == ''">$(AssemblyVersion)</FileVersion>
-    <CommitHash Condition="$(CommitHash) == ''">N/A</CommitHash>
-    <InformationalVersion  Condition="$(InformationalVersion) == ''">$(Version) Commit hash: $(CommitHash)</InformationalVersion>
```

Comment:
This happens automatically now!

Reply:
What is the exact informational version you see with the changes? The above is consistent with what we do for the other projects.

Reply:
It's now `1.5.0-20210820.11+f21362df7b9444d81ba93f8433d3e6a988c604a5`, which is auto-set by the .NET SDK if `$(SourceRevisionId)` is present. This seems to come from the SourceLink package we're using. I'm double-checking this now just so I fully understand.

Looking at all .NET packages, this seems to be the standard format now (adding `+`, then the revision).

https://github.com/dotnet/sdk/blob/main/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets#L58-L69

Reply:
Yep -- `SourceRevisionId` gets set here, which comes from the SourceLink package -- https://github.com/dotnet/sourcelink/blob/main/src/Microsoft.Build.Tasks.Git/build/Microsoft.Build.Tasks.Git.targets#L25-L34.